src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
@rcutmore I think that will just prevent us from opening the options pages in incognito mode. This is a regression, but I don't think it is important enough to block this. We'd still be able to change settings for the incognito pb in the popup.